rpc/core/types/responses.go

```diff
@@ -81,6 +95,8 @@ type SyncInfo struct {
 	EarliestBlockHeight int64          `json:"earliest_block_height"`
 	EarliestBlockTime   time.Time      `json:"earliest_block_time"`
 
+	MaxPeerBlockHeight int64 `json:"max_peer_block_height"`
```

Comment:
Do we need to be backporting this change?

Reply:
Same situation as https://github.com/tendermint/tendermint/pull/6955#discussion_r714861089. If you say it's not relevant, I can drop it.